docs/examples/notebooks/02-set_neuron_params.ipynb

```diff
@@ -0,0 +1,281 @@
+    "    input_node = nengo.Node(\n",
+    "        output=lambda t: [np.sin(t*np.pi),\n",
+    "                          np.cos(t*np.pi)])\n",
+    "\n",
+    "    # Create the FPGA ensemble\n",
+    "    fpga_ens = FpgaPesEnsembleNetwork(\n",
+    "        board,\n",
+    "        n_neurons=4,\n",
+    "        dimensions=2,\n",
+    "        learning_rate=0,\n",
+    "        function=lambda x: [0, 0]  # Initialize decoders to 0\n",
+    "        )\n",
+    "    nengo.Connection(input_node, fpga_ens.input)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Add Probes to Collect Data\n",
+    "\n",
+    "Just so we can monitor the input and output to confirm we set the weights correctly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with model:\n",
+    "    # The original input\n",
+    "    input_p = nengo.Probe(input_node, synapse=0.01)\n",
+    "\n",
+    "    # The output from the FPGA ensemble\n",
+    "    # (filtered with a 10ms post-synaptic filter)\n",
+    "    output_p = nengo.Probe(fpga_ens.output, synapse=0.01)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6: Test Before Setting Weights\n",
+    "\n",
+    "In order to ensure we are in fact setting the weights, we will run the network with the zero-initialized decoders to see what happens first."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "with nengo_fpga.Simulator(model) as sim:\n",
+    "    sim.run(2)  # Run for 2 seconds"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure(figsize=(16, 5))\n",
+    "plt.subplot(1, 2, 1)\n",
+    "plt.title(\"Probed Results (Dimension 1: sin)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 0])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 0])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\")\n",
+    "\n",
+    "plt.subplot(1, 2, 2)\n",
+    "plt.title(\"Probed Results (Dimension 2: cos)\")\n",
+    "plt.plot(sim.trange(), sim.data[input_p][:, 1])\n",
+    "plt.plot(sim.trange(), sim.data[output_p][:, 1])\n",
+    "plt.legend((\"Input\", \"Output\"), loc='lower left')\n",
+    "plt.xlabel(\"Sim time (s)\");"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Notice that the output for both dimensions is zero, as expected."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 7: Set Weights and Test\n",
+    "\n",
+    "Now we will use the values we generated in [Step 3](#Step-3:-Create-Some-Weights) in our FPGA network and test again to show the desired neuron parameters have been set correctly."
```

Comment:
Now, we will configure out FPGA ensemble to use the neuron parameters generated in Step 3. In order to demonstrate the correct behaviour of the network, a new Nengo simulation is created and run.